packages/editor/src/components/post-taxonomies/flat-term-selector.js

```diff
@@ -259,22 +326,55 @@ class FlatTermSelector extends Component {
 			_x( 'Remove %s', 'term' ),
 			singularName
 		);
+		const tagsButtonDescription = _x(
+			'Choose from the most used tags.',
```

Comment:
Worth noting in WordPress this string is the `choose_from_most_used` taxonomy label. Meaning that custom taxonomies can provide their own label.  The term "tags" won't be appropriate for all kind of non-hierarchical taxonomies. Instead of hardcoding the label, this should get the taxonomy labels. See for example newTermLabel, , singularName, termAddedLabel, etc.